src/main/java/com/bravo/user/dao/model/Address.java

```diff
@@ -0,0 +1,66 @@
+package com.bravo.user.dao.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import com.bravo.user.model.dto.AddressReadDto;
+
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "address")
+public class Address {
+
+  @Id
+  @Column(name = "id")
+  private String id;
```

Comment:
When name is specified, there is the potential for typos.  This type of bug would only be caught at runtime or by a unit test.